src/Library/src/Quantisation.cpp

```diff
@@ -39,6 +39,10 @@ const Array2D adjust_quant_indices(const Array2D& qIndices, const int qMatrix) {
 
 const int quant_factor(int q) {
   // Lookup table valid q<120. For q>= 120 quant_factor(q) requires more than 32 bits.
+  if (q>119) {
+    throw std::logic_error("quantization index exceeds 119, corresponding quantization factor exceeds 32 bits.");
```

Comment:
You can safely omit the "corresponding quantization factor exceeds 32 bits." bit as this is merely an arbitrary internal reason for choosing this limit.